app/page/template/meta.htm

```diff
@@ -3,7 +3,7 @@
 <meta name="geoip" country="{{country}}">
 <meta property="wire:version" version="{{config.CURRENT_VERSION_ID}}">
 
-<link href="/image/favicon.ico" rel="shortcut icon" sizes="48x48">
+<link href="/image/favicon.ico" type="image/x-icon" rel="shortcut icon" sizes="48x48">
```

Comment:
It seems to be a good idea to add the type here to indicate the browser the mime type of the file 
https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/The_head_metadata_in_HTML#Adding_custom_icons_to_your_site